app/utils/form/validate-decimals.ts

```diff
@@ -0,0 +1,10 @@
+import BigNumber from 'bignumber.js';
+
+export function validateDecimalPrecision(allowedPrecision: number) {
```

Comment:
Maybe add a TSDoc to this - took me a minute to figure out what this does (although the function name is well descriptive)